devlib/target.py

```diff
@@ -1035,6 +1035,18 @@ def dump_logcat(self, filepath, filter=None, append=False, timeout=30):  # pylin
     def clear_logcat(self):
         adb_command(self.adb_name, 'logcat -c', timeout=30)
 
+    def adb_reboot_bootloader(self):
+        adb_command(self.adb_name, 'reboot-bootloader', timeout=30)
```

Comment:
I wanted to keep the API consistent with what we already have, `clear_logcat` does not expose such a timeout... but probably it makes more sense for the reboot command to expose a timeout.

Ok, let's have it as a parameter.